dev/deps/spark-deps-hadoop-3.2

```diff
@@ -124,6 +134,7 @@ jline-2.14.6.jar
 joda-time-2.9.3.jar
 jodd-core-3.5.2.jar
 jpam-1.1.jar
+json-1.8.jar
```

Comment:
This isn't org.json:json is it? I don't think so given its version, but wanted to check. As you noted earlier and below, org.json:json is Cat X, so just double-checking this is something else?

Reply:
No. It's `com.tdunning:json:1.8`:
https://github.com/apache/hive/blob/rel/release-2.3.4/pom.xml#L646-L650
https://mvnrepository.com/artifact/com.tdunning/json/1.8